Targets/UserInterface/Sources/Login/LoginViewModel.swift

```diff
@@ -26,6 +26,13 @@ class LoginViewModel: ObservableObject {
 // MARK: Auth
 extension LoginViewModel {
     
+    func getAccessToken() async -> Bool {
+        // TODO: 싱글톤에 저장
+        let response = await modelContainer.loginModel.requestAccessToken()
+        return (response != nil)
+        
```

Comment:
```suggestion
        await modelContainer.loginModel.requestAccessToken()
```
위에 Comment on lines +47 to +73 제안사항이 반영된다면 함께 수정될 부분입니다

Reply:
@keeplo  아 이렇게 코드 수정을 제안 받을 수 있네요. 피드백 감사합니다~! :)